### ISSUE TITLE
1.19.3 compatibility

### ISSUE THREAD

**Issue by SlimeDog** [InsightsPlugin/Insights#187]
Spigot 1.19.3-36
Insights 6.13.1-b489

On player connect, errors repeated continuously near where the player is standing. For example (player at 135,4,134; so chunk 8,8):
```
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 7, 7
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 8, 8
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 9, 9
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 7, 8
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 8, 9
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 9, 7
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 8, 7
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 9, 8
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 7, 9
[08:36:30] [Insights-worker-2/WARN]: [Insights] Error occurred while scanning world_lobby @ 8, 8
[08:36:30] [Insights-worker-1/WARN]: [Insights] Error occurred while scanning world_lobby @ 7, 8
[08:36:30] [Insights-worker-2/WARN]: [Insights] Error occurred while scanning world_lobby @ 8, 9
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 9, 9
[08:36:30] [Insights-worker-7/WARN]: [Insights] Error occurred while scanning world_lobby @ 8, 7
[08:36:30] [Insights-worker-1/WARN]: [Insights] Error occurred while scanning world_lobby @ 9, 7
[08:36:30] [Insights-worker-2/WARN]: [Insights] Error occurred while scanning world_lobby @ 9, 8
[08:36:30] [Insights-worker-4/WARN]: [Insights] Error occurred while scanning world_lobby @ 7, 7
[08:36:30] [Insights-worker-6/WARN]: [Insights] Error occurred while scanning world_lobby @ 7, 9
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 7, 7
[08:36:30] [Insights-worker-7/WARN]: [Insights] Error occurred while scanning world_lobby @ 8, 8
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 7, 8
[08:36:30] [Insights-worker-7/WARN]: [Insights] Error occurred while scanning world_lobby @ 8, 9
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 9, 7
[08:36:30] [Insights-worker-7/WARN]: [Insights] Error occurred while scanning world_lobby @ 8, 7
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 9, 8
[08:36:30] [Insights-worker-7/WARN]: [Insights] Error occurred while scanning world_lobby @ 7, 9
[08:36:30] [Insights-worker-1/WARN]: [Insights] Error occurred while scanning world_lobby @ 9, 9
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 7, 7
[08:36:30] [Insights-worker-2/WARN]: [Insights] Error occurred while scanning world_lobby @ 8, 8
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 9, 9
[08:36:30] [Insights-worker-2/WARN]: [Insights] Error occurred while scanning world_lobby @ 7, 8
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 8, 9
[08:36:30] [Insights-worker-2/WARN]: [Insights] Error occurred while scanning world_lobby @ 9, 7
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 8, 7
[08:36:30] [Insights-worker-2/WARN]: [Insights] Error occurred while scanning world_lobby @ 9, 8
[08:36:30] [Insights-worker-0/WARN]: [Insights] Error occurred while scanning world_lobby @ 7, 9
[08:36:31] [Insights-worker-2/WARN]: [Insights] Error occurred while scanning world_lobby @ 7, 7
```

**Comment by mateusneresrb** [comment]
#188

**Comment by rudde0** [comment]
> #188

Could you share jar?

**Comment by FrankHeijden** [comment]
Sorry for the delay, releasing today!

**Comment by SlimeDog** [comment]
Confirmed fixed with build 494 on Spigot 1.19.3-3633 and Paper 1.19.3-365. Thanks.
